### ISSUE TITLE
What model was used in the paper ?

### ISSUE THREAD

**Issue by Mohamed-Dhouib** [hasanar1f/HiRED#2]
Hello, in the paper, you said that you are using LLaVA-Next-7B; however, there is no 7B version of LLaVA-Net. Can you refer me to the Hugging Face page of the model used and provide a `.sh` script to reproduce the paper's results?

**Comment by hasanar1f** [comment]
Thanks for reaching out! The pre-trained checkpoints can be found under [this huggingface model collection](https://huggingface.co/collections/llava-hf/llava-next-65f75c4afac77fd37dbbe6cf). 

Please refer to [these scripts](https://github.com/hasanar1f/HiRED/tree/main/accuracy_benchmarks) and [lmms-eval documentation](https://github.com/EvolvingLMMs-Lab/lmms-eval) to reproduce our results.

**Comment by Mohamed-Dhouib** [comment]
So the llava next you are referring to in the paper is llava 1.6 ? If so which one please ?

**Comment by hasanar1f** [comment]
Yes. We have used llava-hf/llava-v1.6-vicuna-7b-hf and llava-hf/llava-v1.6-vicuna-13b-hf. Thanks for bringing it up. I will clarify this in the paper.

**Comment by Mohamed-Dhouib** [comment]
Ok thanks, I'm trying to implement your work on Llava Next to compare with mine, can you refer me to the function that takes the attention scores per crop and returns the unpruned set of tokens ?

**Comment by hasanar1f** [comment]
Please see the [line 978-979 from here](https://github.com/hasanar1f/HiRED/blob/5d692d9c63a1b6b4a2f7f27a78147493c81fa052/transformers/src/transformers/models/llava_next/modeling_llava_next.py#L978) and the [ViT_Attn_Hook function](https://github.com/hasanar1f/HiRED/blob/5d692d9c63a1b6b4a2f7f27a78147493c81fa052/transformers/src/transformers/models/llava_next/modeling_llava_next.py#L48C7-L48C20). Here, we get the attention of top and bottom ViT layer.

using that attention, we calculated the mask. And finally apply that mask [here](https://github.com/hasanar1f/HiRED/blob/5d692d9c63a1b6b4a2f7f27a78147493c81fa052/transformers/src/transformers/models/llava_next/modeling_llava_next.py#L1015). The unpruned set of tokens is referred to as _image_features_ before it goes into the _pack_HiRED_image_features_ function.

**Comment by Mohamed-Dhouib** [comment]
OK thanks !
